hdr_schemata/utils/create_markdown.py

```diff
@@ -157,6 +160,8 @@ def form_structure(data, form, parent=None):
                         info = {**title, **t_sch["anyOf"][0]}
                     else:
                         info = t_sch
+                elif issubclass(t, BaseModel):
```

Comment:
Removes fields where the type is a pydantic type we have defined like "Organisation".